### ISSUE TITLE
Cycle editor instruction : "arm action"

### ISSUE THREAD

**Issue by Jeff0S** [reaper-oss/sws#571]
_From [Renaud...@gmail.com](https://code.google.com/u/114325195493997457890/) on May 08, 2013 00:04:42_

Dunno if possible but it would be awesome to have an instruction to make an action be a smart tool (exactly like when right click on a toolbar icon)

Exemple of CA

Arm.  
Split item at edit cursor
Step
Unarm
Split item at edit cursor

Or only the first step if the C.A can be define as a toggle ( in left column)

In short, a simple way to trigger a smart tool by keyboard shortcut, left click on icon or mouse modifier

Thanks.

And feel free to delete this FR if undoable due to API or " stuff that only programmers understand" :)

_Original issue: http://code.google.com/p/sws-extension/issues/detail?id=571_
